helpers/constants.ts

```diff
@@ -133,6 +156,8 @@ export const DEFAULT_NAMED_ACCOUNTS = {
   },
   emergencyAdmin: {
     default: 0,
+    1: EMERGENCY_ADMIN[eEthereumNetwork.main],
+    31337: process.env.FORK == "main" ? ETHEREUM_SHORT_EXECUTOR : 1,
```

Comment:
In the case of a mainnet fork, there is no need to transfer ownerships then?
What about forks of other networks? How does the process look like?

Reply:
this is a way to enforce fork mainnets to use the ethereum short executor, maybe should be enforced by market config instead of named accounts

Reply:
then, for main fork there is no need to transfer ownership because its using the correct owner from the beginning ?

Reply:
Removed this abstraction due it also fails in older scenarios. All roles must be deployer until the end of the deployment to not break testnet scripts with reserves.